src/helper/logger.ts

```diff
@@ -36,8 +48,8 @@ export class LoggerStream {
 	}
 }
 
-if (process.env.NODE_ENV === 'production') {
-	logger.debug('Logging initialized at production level');
+if (isProductionMode) {
+	console.log('Logging initialized at production level');
```

Comment:
Why `console.log` instead of the logger?

Reply:
The logger is not defined on this position.

Reply:
check 👍   but can https://github.com/hpi-schul-cloud/node-notification-service/blob/43f2a4d537970a5b9adddd30af660c9d24d6cbd2/src/app.ts#L57 use the logger